Doc/library/socket.rst

```diff
@@ -200,6 +200,23 @@ created.  Socket addresses are represented as follows:
 
   .. versionadded:: 3.8
 
+- :const:`IPPROTO_UDPLITE` is a variant on UDP which allows you to specify
+  what portion of a packet to cover with the checksum. It adds the methods
```

Comment:
```suggestion
  what portion of a packet is covered with the checksum. It adds the methods
```
Not sure about the proposed text but it looks a little more coherent to me.

Reply:
Fixed